src/pages/components/Navbar.tsx

```diff
@@ -106,7 +107,7 @@ export function Navbar() {
       <div className="">
         <ul
           className={`menu menu-vertical fixed top-0 left-0 z-30 flex h-screen h-[100svh] max-h-screen w-full justify-start justify-items-start bg-black pt-[88px] pb-12 text-lg text-white transition-opacity duration-300 lg:hidden${
-            isOpen ? ' opacity-100' : ' pointer-events-none opacity-0'
+            isOpen ? ' opacity-100' : 'pointer-events-none hidden opacity-0'
```

Comment:
Avoid items in the overlay being read by screen reader if they are not rendered, by adding `hidden` class
![](https://user-images.githubusercontent.com/8603085/191321627-f68572a3-5729-48ea-bc93-34058cc84c8e.png)